spec/index.bs

```diff
@@ -7360,12 +7337,12 @@ GPUQueue includes GPUObjectBase;
             If any of the following requirements are unmet, throw an {{OperationError}} and stop.
             <div class=validusage>
                 - |copySize|.[=Extent3D/depthOrArrayLayers=] must be `1`.
-                - |destination|.{{GPUImageCopyTexture/texture}}.{{GPUTexture/[[textureUsage]]}}
+                - |destination|.{{GPUImageCopyTexture/texture}}.{{GPUTexture/[[descriptor]]}}.{{GPUTextureDescriptor/usage}}
```

Comment:
in algorithms like this, it would make more sense to just have the |descriptor| be an argument to the algorithm, instead of repeatedly extracting it from |destination|

Reply:
I read through all changes, this is the only place where the algorithm doesn't use |destination| except its texture's descriptor info. At other places, we need to use |destination| itself. However, I feel like this algorithm is not finalized yet. For example, it may need to add 
 - [$validating GPUImageCopyTexture$](|destination|, |copySize|) returns `true`,
 during validation. 

It may need to say that 
 - copy contents into |destination|. 
after validation.

In short, I think |destination| is a must-to-have argument. But the argument |descriptor| you proposed is more like a implementation detail. If you really want to shorten the text, maybe we can add a line at the beginning of the validation as follows: 
 - Let |descriptor| be |destination|.{{GPUImageCopyTexture/texture}}.{{GPUTexture/[[descriptor]].

WDYT? (I feel like it is a temp variable and it is a implementation detail. not sure whether it is needed. )

Reply:
Done. Right now the text is simpler and cleaner than before. 

Please help to merge this PR if you have no other concerns.